### ISSUE TITLE
Dynamic DNS - OpenDNS Missing since 19.7.7

### ISSUE THREAD

**Issue by DanMc85** [opnsense/plugins#1634]
OpenDNS option is missing from the Dynamic DNS since 19.7.7.

It made existing entries error out and it is no longer in the drop down list to re-select.

Can this code be put back into Dynamic DNS?

It allowed for OpenDNS's paid IP filtering DNS service to always work if a Dynamic Public IP changed on the OPNsense Fireewall.

https://support.opendns.com/hc/en-us/articles/227987767-Using-Dynamic-DNS-with-OpenDNS

**Comment by fichtner** [comment]
It was included in core.

**Comment by DanMc85** [comment]
> It was included in core.

Yes OpenDNS has its own menu option... However, this requires use of OPNsense as DNS server.  I do not use the OPNsense DNS server, I have my own Windows Server 2019 DNS server which uses the OpenDNS IP addresses as forwarders.  Thus it makes the core option useless in this situation. The Dynamic DNS option is what is desired.  

Unless there is a way to use the core option without it taking over DNS but still have it update the IP on the OpenDNS side of things.
